e2e/tests/ui-driven/src/helpers/addComponent.ts

```diff
@@ -153,6 +157,21 @@ export const createQuestionWithOptions = async (
   );
 };
 
+export const createQuestionWithDataFieldOptions = async (
```

Comment:
Would appreciate feedback here, the `createBaseComponent` seems reliant on the `option: string[]` and my attempts to add in an option which contains a data value pulled on a lot of threads. These below lines seemed the easiest, lowest impact additions. 

Is there a pattern I should be using here with discriminated unions and overloaded functions in Typescript?

Reply:
I'm fine with this approach of a new helper `createQuestionWithDataFieldOptions` - seems pragmatic and correct for moving this forward :+1: